### PR TITLE
Fix typos in release notes ert 9.0

### PR DESCRIPTION
**Pull request by hnformentin**
Fix typos observed while reading the release notes.

### CODE REVIEW

File: docs/about/release_notes.rst
Comment (berland) [inline docs/about/release_notes.rst:38]: analyses is here plural of analysis

**Review by oyvindeide — APPROVED** [review]
Thank for for you contribution!

**Comment by hnformentin** [comment]
I can not merge the issue with "Only those with write access to this repository can merge pull requests."

**Pull request merged**
